db/migrate/20190517174550_add_user_to_post.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddUserToPost < ActiveRecord::Migration[5.2]
+  def change
+    add_reference :posts, :user, foreign_key: true
```

Comment:
You can add the foreign key in the same migration that you create the post. Make it just one migration and delete this file.